dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -271,26 +271,22 @@ DSRA4011.tran.none.iso.switch.unsupported=DSRA4011E: The JDBC driver does not su
 DSRA4011.tran.none.iso.switch.unsupported.explanation=The JDBC specification does not support switching to an isolation level of TRANSACTION_NONE.
 DSRA4011.tran.none.iso.switch.unsupported.useraction=Avoid switching to an isolation level of TRANSACTION_NONE.
 
-DSRA4012.ignored.datasource.config.used=DSRA4012I: When using Oracle UCP the following data source properties are ignored: statementCacheSize and validationTimeout. Use the equivalent Oracle UCP functionality.
-DSRA4012.ignored.datasource.config.used.explanation=The statementCacheSize and validationTimeout data source properties are not valid when using Oracle UCP.
-DSRA4012.ignored.datasource.config.used.useraction=Remove the data source properties and instead use the equivalent Oracle UCP functionality.
+DSRA4012.ignored.datasource.config.used=DSRA4012I: When using Oracle UCP the following data source properties are ignored: {0}.
+DSRA4012.ignored.datasource.config.used.explanation=The listed data source properties are not valid when using Oracle UCP.
+DSRA4012.ignored.datasource.config.used.useraction=Remove the data source properties and instead use the equivalent Oracle UCP functionality, if available.
```

Comment:
Change 
"Remove the data source properties and instead use the equivalent Oracle UCP functionality, if available."
To
"Remove the listed data source properties and instead use the equivalent Oracle UCP functionality, if available."

is there any information we can give about action to take if no UCP functionality is available?

Reply:
Ok, I'll make that update.  There isn't really any action they can take in the case that they're trying to use function available in Liberty's connection pool but not Oracle's.

Reply:
Hi alexsm82- the word "invalid" is against guidance. We usually use "not valid" instead but that doesn't  work in this case.

Will this user action message appear in the same panel as the list of data source properties? Or is it something that a user would look up to get more information about the initial error message?

Reply:
Hi @dmuelle. Only the message itself appears in the logs.  The user would have to search to find the explanation or user action - like here: https://www.ibm.com/support/knowledgecenter/SSAW57_8.5.5/com.ibm.websphere.messages.doc/com.ibm.ws.rsadapter.resources.IBMDataStoreAdapterNLS.html.

Since we shouldn't use invalid, what about:
DSRA4013.ignored.connection.manager.config.used.useraction=Remove the connection manager properties that are not valid and instead use the equivalent Oracle UCP functionality, if available.

Reply:
@alexsm82 I think that's a good solution

"Remove the connection manager properties that are not valid and instead use the equivalent Oracle UCP functionality, if available."

Once the changes are made I'll mark it ID reviewed